spell/spell-reviewer-mainnet-checklist.md

```diff
@@ -135,6 +135,11 @@
         * [`DssExecLib.setGlobalDebtCeiling(amount)`](https://github.com/makerdao/dss-exec-lib/blob/v0.0.9/src/DssExecLib.sol#L428)
         * [`DssExecLib.increaseGlobalDebtCeiling(amount)`](https://github.com/makerdao/dss-exec-lib/blob/v0.0.9/src/DssExecLib.sol#L436)
         * [`DssExecLib.decreaseGlobalDebtCeiling(amount)`](https://github.com/makerdao/dss-exec-lib/blob/v0.0.9/src/DssExecLib.sol#L445C14-L445C39)
+* IF additional dependencies (i.e. `./src/dependencies/` directory) are present:
+  * [ ] IF the dependencies contracts/libraries have been audited
+    * [ ] FOREACH contract/library ensure its content matches (i.e. diff check) the latest audited version
```

Comment:
Let's not introduce a new FOREACH pattern in this PR, as all checks in all checklists should be applied for each occurrence of the checked item anyway. If we want to change this implicit pattern into explicit keyword, I would suggest to introduce it to all checklists in a separate PR. 

Here you can just say 

```suggestion
    * [ ] Each contract/library exactly matches (i.e. diff check) source code of the latest audited version
      _List each dependency with the link to its source code here_
```